docs/mllib-dimensionality-reduction.md

```diff
@@ -218,6 +238,30 @@ public class PCA {
 {% endhighlight %}
 
 </div>
+
+<div data-lang="python" markdown="1">
+
+The following code demonstrates how to compute principal components on a `RowMatrix`
+and use them to project the vectors into a low-dimensional space.
+
+{% highlight python %}
+from pyspark.mllib.linalg import Matrix
+from pyspark.mllib.linalg.distributed import RowMatrix
+from numpy.random import RandomState
+
+# Generate random data with 50 samples and 30 features.
+rng = RandomState(0)
+mat = sc.parallelize(rng.randn(50, 30))
+rm = RowMatrix(mat)
```

Comment:
Minor: I might change `mat` here to `data`, and then change `rm` to `mat` just to be consistent.